_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -69574,6 +69577,13 @@
 	},
 /turf/open/floor/iron/white,
 /area/station/science/robotics/lab)
+"wjJ" = (
+/obj/item/pen{
+	pixel_y = 4;
+	pixel_x = 5
+	},
+/turf/open/genturf,
+/area/icemoon/underground/unexplored/rivers/deep)
```

Comment:
This is at approximately (101, 114)  (105, 118) on the bottom z of Icebox. It is a random pen that has somehow managed to creep into a genturf area

Reply:
I used Find On Map and i still can't find it, what the fuck

Reply:
I've manually edited it in code as i cannot for the life of me find it on strongDMM, i HOPE with works